packages/components/src/lib/PinFileRequest.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @class
+ */
+export class PinFileRequest {
+  /**
+   * @param {string} name - The name of the file.
+   * @param {string} [uri] - The URI of the file (optional).
+   * @param {string} [content] - The content of the file (optional).
+   */
```

Comment:
These can be TypeScript as well

Reply:
Ok, I thought I tried that some point, and it caused a build failure. I just tried it now, though, and it worked ok. Could be that I moved it from somewhere else into `lib`, or that I was trying to import it into a different file (e.g. a `vue`) file....